substrate/primitives/runtime/src/lib.rs

```diff
@@ -944,9 +949,13 @@ impl<'a> ::serde::Deserialize<'a> for OpaqueExtrinsic {
 	}
 }
 
+// TODO: OpaqueExtrinsics cannot act like regular extrinsics, right?!
```

Comment:
This goes back to the question of who is using `trait Extrinsic`, and do they actually care about it being bare or not? 

`OpaqueExtrinsic` is the client's view of an extrinsic, which is purely a an opaque blob, with no understanding of its _format_ or _preamble_, as the new terminology as. So in principle, this function should be non-existent. 

I hope your investigation will confirm that this trait can be removed, or that this function can be removed from it.

Reply:
Just chiming in to say that I've also questioned the usefuleness of the `Extrinsic` trait when working with UTXO based runtimes.

Here is what I've done with it in Tuxedo:

https://github.com/Off-Narrative-Labs/Tuxedo/blob/9ae61fdf1e81df21a2fdea9fbf26872ee512e4ad/tuxedo-core/src/types.rs#L102-L139